--- lua-5.2.0/src/Makefile	2011-09-19 14:45:14.000000000 +0200
+++ lua-5.2.0/src/Makefile.dynlib	2012-06-02 12:36:54.854704843 +0200
@@ -28,6 +28,7 @@
 
 PLATS= aix ansi bsd freebsd generic linux macosx mingw posix solaris
 
+LUA_SO= liblua.so.$(V)
 LUA_A=	liblua.a
 CORE_O=	lapi.o lcode.o lctype.o ldebug.o ldo.o ldump.o lfunc.o lgc.o llex.o \
 	lmem.o lobject.o lopcodes.o lparser.o lstate.o lstring.o ltable.o \
@@ -43,7 +44,7 @@
 LUAC_O=	luac.o
 
 ALL_O= $(BASE_O) $(LUA_O) $(LUAC_O)
-ALL_T= $(LUA_A) $(LUA_T) $(LUAC_T)
+ALL_T= $(LUA_A) $(LUA_T) $(LUAC_T) $(LUA_SO)
 ALL_A= $(LUA_A)
 
 # Targets start here.
@@ -59,6 +60,11 @@
 	$(AR) $@ $(BASE_O)
 	$(RANLIB) $@
 
+$(LUA_SO): $(BASE_O)
+	gcc -Wl,-soname=liblua.so.$(V) -shared -o $@ $(MYLDFLAGS) $^ $(LIBS)
+	rm -f liblua.so
+	ln -s $@ liblua.so
+
 $(LUA_T): $(LUA_O) $(LUA_A)
 	$(CC) -o $@ $(LDFLAGS) $(LUA_O) $(LUA_A) $(LIBS)
 
