--- lua-5.2.0/src/luaconf.h	2011-12-06 17:58:36.000000000 +0100
+++ lua-5.2.0/src/luaconf.h.modules	2012-06-02 13:27:58.554648459 +0200
@@ -100,14 +100,17 @@
 #else			/* }{ */
 
 #define LUA_VDIR	LUA_VERSION_MAJOR "." LUA_VERSION_MINOR "/"
-#define LUA_ROOT	"/usr/local/"
+#define LUA_ROOT	"/usr/"
 #define LUA_LDIR	LUA_ROOT "share/lua/" LUA_VDIR
 #define LUA_CDIR	LUA_ROOT "lib/lua/" LUA_VDIR
+#define LUA_CDIR64	LUA_ROOT "lib64/lua/" LUA_VDIR
 #define LUA_PATH_DEFAULT  \
 		LUA_LDIR"?.lua;"  LUA_LDIR"?/init.lua;" \
-		LUA_CDIR"?.lua;"  LUA_CDIR"?/init.lua;" "./?.lua"
+		LUA_CDIR"?.lua;"  LUA_CDIR"?/init.lua;" \
+		LUA_CDIR64"?.lua;" LUA_CDIR64"?/init.lua;" "./?.lua"
 #define LUA_CPATH_DEFAULT \
-		LUA_CDIR"?.so;" LUA_CDIR"loadall.so;" "./?.so"
+		LUA_CDIR"?.so;" LUA_CDIR"loadall.so;" \
+		LUA_CDIR64"?.so;" LUA_CDIR64"loadall.so;" "./?.so"
 #endif			/* } */
 
 
