--- lua-5.3.1/src/luaconf.h.0002~	2015-07-16 18:49:32.567567492 +0200
+++ lua-5.3.1/src/luaconf.h	2015-07-17 08:37:26.644707936 +0200
@@ -286,6 +286,11 @@
 ** ===================================================================
 */
 
+#ifndef LUA_NO_COMPAT
+#define LUA_COMPAT_5_2 1
+#define LUA_COMPAT_5_1 1
+#endif
+
 /*
 @@ LUA_COMPAT_5_2 controls other macros for compatibility with Lua 5.2.
 @@ LUA_COMPAT_5_1 controls other macros for compatibility with Lua 5.1.
